<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>확률 계산기</string>
  </property>
  <widget class="QTextEdit" name="txt_pro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_want">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_do">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>출현확률 (%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>시행 횟수 (회)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>110</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>희망 출현 횟수 (회)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cal">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>50</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>계 산</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>380</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_acc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>380</width>
     <height>40</height>
    </rect>
   </property>
   <property name="title">
    <string>정확도 설정</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QRadioButton" name="rb_acc1">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>15</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>낮음</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_acc2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>15</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>보통</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_acc3">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>15</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>높음</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
